packages/@lwc/ssr-runtime/package.json

```diff
@@ -43,7 +43,7 @@
             }
         }
     },
-    "dependencies": {
+    "devDependencies": {
```

Comment:
Making this a devDep means that it's bundled into `ssr-runtime` which just makes things simpler (e.g. swapping packages in Tachometer tests).